Mathlib/Topology/Basic.lean

```diff
@@ -1356,6 +1356,26 @@ theorem mem_closure_iff_nhds_basis {a : α} {p : ι → Prop} {s : ι → Set α
     simp only [Set.Nonempty, mem_inter_iff, exists_prop, and_comm]
 #align mem_closure_iff_nhds_basis mem_closure_iff_nhds_basis
 
+theorem clusterPt_iff_forall_mem_closure [TopologicalSpace X] {F : Filter X} {x : X} :
```

Comment:
For consistency above and below:
```suggestion
theorem clusterPt_iff_forall_mem_closure {F : Filter α} {x : α} :
```
and similarly with the other three lemmas you're introducing here